### PR TITLE
Overcoming Verticality - Adds ramps, extends the medical lift and fixes stair gravity.

### PR DESCRIPTION
**Pull request by 1Sparky1**
Adds ramps, which function as dummy verticality like many of the stairs, except they don't trigger the "Stairs are dangerous" response, allowing certain areas to be more accessible.

Extends the medical lift to deck three, allowing a non-stair based way to reach the upper department.

Fixes the "Stairs are dangerous" interaction not checking for gravity.

### CODE REVIEW

File: code/modules/multiz/structures.dm [hunk 1]
```diff
@@ -376,9 +376,10 @@
 	anchored = TRUE
 	icon = 'icons/obj/structure/stairs.dmi'
 	icon_state = "np_stair"
+	var/steps = TRUE
```
Comment (FluffyGhoster) [inline code/modules/multiz/structures.dm:379]: DMDoc this

File: code/modules/multiz/structures.dm [hunk 2]
```diff
@@ -400,6 +401,18 @@
 /obj/structure/platform_stairs/full/south_north_cap/half
 	icon_state = "p_stair_sn_half_cap"
 
+/obj/structure/platform_stairs/ramp
+	name = "ramp"
+	desc = "An smooth yet archaic form of locomotion along the Z-axis."
```
Comment (FluffyGhoster) [inline code/modules/multiz/structures.dm:406]:
```suggestion
	desc = "An smooth yet archaic form of locomotion along the vertical axis."
```

File: code/modules/multiz/structures.dm [hunk 3]
```diff
@@ -400,6 +401,18 @@
...
+	layer = RUNE_LAYER
```
Comment (FluffyGhoster) [inline code/modules/multiz/structures.dm:407]: This isn't a rune

**Comment by DreamySkrell** [comment]
I think the medical lift is fine.

I'm really not a fan of the ramps, I don't think the sprite looks nice visually.
And a lot of the areas that get ramps now, should not have had stairs or the fake verticality in the first place, like the morgue or machinist workshop.

**Comment by 1Sparky1** [comment]
> I think the medical lift is fine.
> 
> I'm really not a fan of the ramps, I don't think the sprite looks nice visually. And a lot of the areas that get ramps now, should not have had stairs or the fake verticality in the first place, like the morgue or machinist workshop.

I disagree on the areas not having verticality. I think it makes a single z-level feel more dynamic and interesting. I have, however, seen several mentions of the sprites appearances, so I'm going to ask nicely to see if a spriter will give it a go. If not, I'll scrap them and just make it all flat.

**Review by FluffyGhoster — APPROVED** [review]
Code LGTM

File: code/modules/multiz/structures.dm [hunk 4]
```diff
@@ -400,6 +404,18 @@
 /obj/structure/platform_stairs/full/south_north_cap/half
 	icon_state = "p_stair_sn_half_cap"
 
+/obj/structure/platform_stairs/ramp
+	name = "ramp"
+	desc = "An smooth yet archaic form of locomotion along the vertical axis."
```
Comment (NonQueueingMatt) [inline code/modules/multiz/structures.dm:409]:
```suggestion
	desc = "A smooth yet archaic form of locomotion along the vertical axis."
```

**Review by NonQueueingMatt — APPROVED** [review]
we'll see how it goes, I guess. I do like the idea of ramps

**Pull request merged**